keras/layers/wrappers.py

```diff
@@ -122,7 +122,7 @@ def step(x, states):
             # to process batches of any size
             # we can go with reshape-based implementation for performance
             X = K.reshape(X, (-1, ) + input_shape[2:])  # (nb_samples * timesteps, ...)
-            y = self.layer.call(X)  # (nb_samples * timesteps, ...)
+            y = self.layer.call(X, mask)  # (nb_samples * timesteps, ...)
```

Comment:
The proper solution in this case is to add a test `if mask is None` and go with the `rnn`-based solution above in that case.